Force the compiler used by nvcc to use the C++11 standard to avoid:
error: identifier "__float128" is undefined
Author: Davide Vanzo (Vanderbilt University)
diff -ru amber16.orig/AmberTools/src/configure2 amber16/AmberTools/src/configure2
--- amber16.orig/AmberTools/src/configure2	2019-02-28 10:41:07.185332187 -0600
+++ amber16/AmberTools/src/configure2	2019-02-28 11:31:13.125415364 -0600
@@ -1731,9 +1731,9 @@
       pmemd_cu_libs="./cuda/cuda.a -L\$(CUDA_HOME)/lib64 -L\$(CUDA_HOME)/lib -lcurand -lcufft -lcudart $fc_cxx_link_flag"
       pbsa_cu_libs="-L\$(CUDA_HOME)/lib64 -L\$(CUDA_HOME)/lib -lcublas -lcusparse -lcudart -lcuda $fc_cxx_link_flag"
       if [ "$optimise" = 'yes' ]; then
-          nvcc="$nvcc -use_fast_math -O3 "
+          nvcc="$nvcc -use_fast_math -O3  -Xcompiler '-std=c++11'"
       else
-          nvcc="$nvcc -use_fast_math -O0 "
+          nvcc="$nvcc -use_fast_math -O0  -Xcompiler '-std=c++11'"
       fi
 
       if [ "$mpi" = 'yes' ]; then
